.github/workflows/ci_emmocheck.yml

```
@@ -26,4 +26,4 @@ jobs:
 
     - name: Check EMMO
       run: |
-        emmocheck --local --verbose --check-imported crystallography.owl
+        emmocheck --url-from-catalog --verbose crystallography.owl
```

```suggestion
        #emmocheck --url-from-catalog --verbose crystallography.owl
```
I suggest removing emmocheck for now, until support for ttl is included in emmopython